Dockerfile

```diff
@@ -43,18 +45,23 @@ RUN apk update --no-cache && \
 RUN mix local.rebar --force && \
     mix local.hex --force
 
-# Copy dependency config first
+# Install & complie Hex dependencies
```

Comment:
Don’t you think dependencies is enough ? Because sometimes we could have dependencies not from Hex.

Reply:
I wanted to follow the other comments format…

> ` Install … dependencies